conduit/models/erm.py

```diff
@@ -85,7 +80,7 @@ def inference_epoch_end(self, outputs: EPOCH_OUTPUT, stage: Stage) -> MetricDict
             acc1, acc5 = precision_at_k(logits=logits_all, targets=targets_all, top_k=(1, 5))
```

Comment:
You removed the import of `precision_at_k` so I don't understand how this still works

Reply:
Ah, it wouldn't do if it were actually run -- the only models that are tested are the fair ones. As I said, I hope to remove the models from conduit at some point and lay them to rest in a model graveyard repo so I won't be shoring this up.